yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/configs/MultiServerConfig.java

```diff
@@ -88,9 +78,9 @@ public Set<String> getVersionsUnderTest() {
     @Override
     public String toString() {
         if (initialConnection == 0) {
-            return "MultiServer (Embedded then External)";
+            return "MultiServer version " + externalServer.getVersion() + " (Embedded then External)";
```

Comment:
Perhaps a more pithy version, as these go in the test names:
```suggestion
            return "MultiServer (Embedded then " + externalServer.getVersion() + ")";
```

Reply:
Done.